recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.2" %}
```

Comment:
OK, then let me suggest a different way to phrase this:

Feedstocks are primarily a chronological flow of recipe development.  It is difficult to go backwards and support old versions while maintaining coherent history.  For this reason, please submit PRs for feedstocks with the oldest version that you think we should support.  Community members are welcome to request other versions, and are encouraged to add their support towards building out the version space once the staged recipe is merged.

----------

When you say "please do it this way because *I* need it, it comes across as (small) abuse of the conda-forge community effort for your benefit.  That's not true, clearly, because of your contributions and comments here, but it can come across that way.